clippy_lints/src/loops.rs

```diff
@@ -293,18 +293,48 @@ declare_clippy_lint! {
 declare_clippy_lint! {
     /// **What it does:** Checks for empty `loop` expressions.
     ///
-    /// **Why is this bad?** Those busy loops burn CPU cycles without doing
-    /// anything. Think of the environment and either block on something or at least
-    /// make the thread sleep for some microseconds.
+    /// **Why is this bad?** Due to [an LLVM codegen bug](https://github.com/rust-lang/rust/issues/28728)
+    /// these expressions can produce undefined behavior. Also, these busy loops
```

Comment:
```suggestion
    /// these expressions can produce undefined behavior (in release mode). Also, these busy loops
```

Reply:
I would not say 'can produce undefined behavior', I'd say 'can be miscompiled'. The user hasn't done anything wrong, the compiler is interpreting their code incorrectly.

Reply:
I'm good with both or a combination "can be miscompiled, producing undefined behavior"